### DISCUSSION TITLE
More detailed guide for how to sign with azure trusted sign

### DISCUSSION THREAD

**Discussion by kristeroksnes** [ebourg/jsign#244] [Q&A]
Hello! 

I was wondering if its possible to get some help with azure trusted signing. 
I have gotten to the point where jsign prints out "Adding Authenticode signature to "EXE file" " But i doesnt seem to actually sign the EXE. If i then check the properties of the file it has no signatures. 

Do i need anything else than what is said in the example? What could i be doing wrong?  Thanks!

**Comment by ebourg** [comment]
Could add the `--debug` parameter and post the output please (or send it to ebourg@apache.org) ? I'll get a look.

**Reply by kristeroksnes** [reply]
Yep! I emailed you the results after adding --debug. Thanks for you help!
